dts/arm/st/wb/stm32wb.dtsi

```diff
@@ -293,6 +293,25 @@
 			status = "disabled";
 			label = "LPTIM_1";
 		};
+
+		usb: usb@40006800 {
+			compatible = "st,stm32-usb";
+			reg = <0x40006800 0x400>;
```

Comment:
Details pulled from the RM0434 reference manually from ST for register location, interrupts, clock, etc.